app/controllers/stores_controller.rb

```diff
@@ -0,0 +1,16 @@
+class StoresController < ApplicationController
+  def new
+    @store = Store.new
+  end
+
+  def create
+    @store = Store.create_with_user(store_params, current_user)
+    redirect_to dashboard_index_path
```

Comment:
Maybe throw in a flash method here letting them know a store was created.

Reply:
great idea about a flash message.